solutions/observability/cloud/monitor-microsoft-azure-with-elastic-agent.md

```diff
@@ -266,10 +266,10 @@ To add the integration:
     **Storage account key**
     :   A valid access key defined for the storage account.
 
-    :::{image} ../../../images/observability-agent-tut-azure-activity-log-settings.png
-    :alt: Screenshot of integration settings for Azure activity logs
-    :class: screenshot
-    :::
+        :::{image} ../../../images/observability-agent-tut-azure-activity-log-settings.png
+        :alt: Screenshot of integration settings for Azure activity logs
+        :class: screenshot
+        :::
```

Comment:
Screenshot wasn't rendering

Reply:
Looks like the image under subscription ID in line 116 also needs to be fixed.